packages/contracts/test/lib/libErc20.spec.ts

```diff
@@ -11,7 +12,7 @@ import { TestERC20 } from "../../typechain/TestERC20";
 
 const createFixtureLoader = waffle.createFixtureLoader;
 
-describe("LibErc20", function() {
+describe("LibErc20", function () {
```

Comment:
Ideally should be an arrow function. If you need t manipulate things *about* the test (i.e. if not local and you call `evm_mine`) and use `this` then you can leave it as a `function ()` over `() => {}`

Reply:
updating to () => {}